tests/backgroundTestServers.py

```diff
@@ -49,11 +50,35 @@ def runSubjectService(args, isStartedEvent):
     isStartedEvent.set()
 
 
+def runExampleService(args, isStartedEvent):
+    exampleServer = ExampleService(args)
+    exampleThread = threading.Thread(
+        name='exampleThread',
+        target=exampleServer.wsRemoteService.runForever
+    )
+    exampleThread.start()
+    while exampleServer.wsRemoteService.started is False:
+        time.sleep(.1)
+    isStartedEvent.set()
+
+
+defaultCfg = StructDict({'sessionId': "test",
+                         'subjectName': "test_sample",
+                         'subjectNum': 1,
+                         'subjectDay': 1,
+                         'sessionNum': 1})
+
+defaultProjectArgs = StructDict({'config': defaultCfg,
+                                 'mainScript': 'projects/sample/sample.py',
+                                 'port': 8921,
+                                 'test': True})
+
 class BackgroundTestServers:
     def __init__(self):
         self.projectProc = None
         self.dataProc = None
         self.subjectProc = None
+        self.exampleProc = None
         if platform.system() == "Darwin":
```

Comment:
This is older code, but might be nice to have a comment here explaining why this is necessary on Mac